sql/chunk.sql

```diff
@@ -65,6 +65,57 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.dimension_calculate_default_range_closed(
+        dimension_value   BIGINT,
+        num_slices        SMALLINT,
+    OUT range_start  BIGINT,
+    OUT range_end    BIGINT)
+LANGUAGE PLPGSQL STABLE AS
+$BODY$
+DECLARE
+    inter BIGINT;
+BEGIN
+       IF dimension_value < 0 THEN 
+            RAISE 'Dimension values for close dimensions should be positive. Got: %', dimension_value;
+       END IF;
+       inter := (2147483647 / num_slices);
+       IF dimension_value >= inter * (num_slices - 1) THEN
+          --put overflow from integer-division errors in last range
+          range_start = inter * (num_slices - 1);
+          range_end = 2147483647;
```

Comment:
Is there really no way to define constants in PLPGSQL? Having these numbers spread all over is a pain (also knowing what they mean and that they are right).